cloudinit/config/cc_byobu.py

```diff
@@ -39,6 +39,7 @@
     "description": MODULE_DESCRIPTION,
     "distros": distros,
     "frequency": PER_INSTANCE,
+    "activate_by_schema_keys": [],
```

Comment:
I think we can provide "apt_pipelining" key here because we end up NOOP if value is unset which == "os"
https://github.com/canonical/cloud-init/blob/main/cloudinit/config/cc_apt_pipelining.py#L64

Reply:
done